.travis.yml

```diff
@@ -0,0 +1,29 @@
+os:
+  - linux
+sudo: false
+addons:
+  apt:
+    # Flutter depends on /usr/lib/x86_64-linux-gnu/libstdc++.so.6 version GLIBCXX_3.4.18
+    sources:
+      - ubuntu-toolchain-r-test # if we don't specify this, the libstdc++6 we get is the wrong version
+    packages:
+      - libstdc++6
+      - fonts-droid
+env:
+  - FLUTTER_VERSION=beta
```

Comment:
Looks like this env variable is defined twice. Could we please only run against Beta? I've run into various issues with Master and Dev, and it's a bit too much maintenance work to support all these different versions, some of which aren't that stable.